README.md

```diff
@@ -102,6 +102,41 @@ setTimeout(subprocess.cancel, 30000)
 
 It creates a `youtube-dl` using the `binaryPath` provided.
 
+### using youtube-dl's `load-info-json` flag
+
+When download a video with youtube-dl, the page gets downloaded retiving useful information stored in a `YtResponse`.
+This information can be dumped using the `dump-json` or the `dump-single-json` flags.
+Then this info can be piped back to youtube-dl with the `load-info-json` flag, so that the page won't be download again.
+
+```js
+const youtubedl = require('youtube-dl-exec')
+
+const getInfo = (url, flags) => youtubedl(url, { dumpSingleJson: true, ...flags })
+
+const fromInfo = (info, flags) => {
+  const process = youtubedl.exec('', { loadInfoJson: '-', ...flags })
+  process.stdin.write(JSON.stringify(info))
```

Comment:
instead of interacting with `process`, can you just write to a file instead? 

it will be the example easier to understand.